docs/self-hosted-apps/jitsi.md

```diff
@@ -0,0 +1,35 @@
+## What is Jitsi
+As [Jitsi](https://jitsi.github.io/handbook/docs/intro)'s website states:
+
+!!!quote "What is Jitsi?"
+    Jitsi is a collection of Open Source projects which provide state-of-the-art video conferencing capabilities that are secure, easy to use and easy to self-host.
+
+In other words, Jitsi is a video conferencing suite - video server, client, video stream negotiator, media session manager, SIP gateway, and stream/recording manager. This gives the ability to run your own Zoom-like/Google Meet-like/Teams-like meeting with hardware at home.
```

Comment:
Maybe this should live in the quote block above? Seems like you define Jitsi in short, and then in more words immediately after.

Reply:
I've updated the section - the quote was meant as a copy+paste from the Jitsi website, then an interpreted explanation. However I think that you're right that the wording may be confusing, so I've removed the quote altogether and kept the interpreted version (also can remove that an instead just have the quote). Let me know what you think